src/components/developersBigCards/developersBigCards.js

```diff
@@ -3,7 +3,7 @@ import { Card, Accordion, Media, Table } from 'react-bootstrap';
 
 const DevBigCards = ({ data }) => {
   return(
-    <React.Fragment>
+    <>
```

Comment:
What's the reason replace Fragment to <>?

Reply:
Why shouldn't we use <></> ? After all, it is more convenient than <React.Fragment></ React.Fragment> and fragment has no useful features now

Reply:
You're right. We can use both options but yours is shorter, ty.